Appraise/settings.py

```diff
@@ -117,7 +117,7 @@
         INSTALLED_APPS.append('debug_toolbar')
         warnings.warn('Enabled Django Debug Toolbar in installed apps')
     except ImportError:
-        warnings.warn('Django Debug Toolbar not installed')
+        pass

```

Comment:
I know this message is too frequent and quite annoying, but I think it may be better to make life quality improvement changes in a separate PR.